README.md

```diff
@@ -20,6 +20,7 @@ The module name is `opentracing_instrumentation`.
 ### Supported client frameworks
 
 The following libraries are instrumented for tracing in this module:
+ * `boto3`
```

Comment:
let's add a small comment explaining what it is (the other module names are more obvious), and maybe turn it into a link

Reply:
I've turned `boto3` into a link and added a short description.

I can also create an extra PR that will do the same for the rest of the list.
What do you think?